### PR TITLE
fibonacci

### PR DESCRIPTION
**Pull request by kasiasieradzka**

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -3,9 +3,30 @@
...
+
+        return result;
 }
 
 int fibonacci_recursive(int sequence) {
     // TODO: Your implementation goes here
     return 0;
+
+    if (sequence <= 1)
+        return sequence;
```
Comment (Pawellos) [inline module1/homework/fibonacci/fibonacci.hpp:28]: Some people says the good practisie is used to { }, even for one-line instructions.

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -3,9 +3,30 @@
...
+   
+    else
```
Comment (DamiXioR) [inline module1/homework/fibonacci/fibonacci.hpp:30]: i think last else is redundant

Comment (kasiasieradzka) [reply module1/homework/fibonacci/fibonacci.hpp]: Ok I remove it

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**